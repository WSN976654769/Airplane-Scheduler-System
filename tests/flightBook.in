flight bok
flight book
flight 0 book
location import locations.csv
FLIGHT ADD Monday 5:20 Sydney beijing 20
flight 0 bok
flight 0 book
flight 0 book -20 
flight 0 book 200
flight 0 book 200
flight 0 book abs
exit