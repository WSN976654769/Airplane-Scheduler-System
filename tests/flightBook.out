User: Invalid Flight ID.

User: Invalid Flight ID.

User: Invalid Flight ID.

User: Imported 51 locations.

User: Successfully added Flight 0.

User: Flight 0
Departure:    Mon 05:20 Sydney
Arrival:      Mon 17:45 Beijing
Distance:     8,939km
Duration:     12h 25m
Ticket Cost:  $2704.80
Passengers:   0/20

User: Booked 1 passengers on flight 0 for a total cost of $2704.80

User: Invalid number of passengers to book.

User: Booked 19 passengers on flight 0 for a total cost of $48610.25
Flight is now full.

User: Booked 0 passengers on flight 0 for a total cost of $0.00
Flight is now full.

User: Invalid number of passengers to book.

User: Application closed.
